src/deck/drivers/src/Kconfig

```diff
@@ -390,6 +390,37 @@ config DECK_OA
     help
         Codename: Obstacle Avoidance driver, obsolete deck.
 
+config DECK_SERVO
+    bool "Support the servo deck (prototype/via BigQuad deck)"
```

Comment:
This seem to also be able to be driven from the Crazyflie Vcom/Tx directly right?

Reply:
Yes, the default is TX**2**, here a photo of a working setup:
![image](https://github.com/bitcraze/crazyflie-firmware/assets/10943061/6e728bd6-3086-4264-be7f-ffc0329e20f8)

Reply:
I meant that you would typically use the prototype deck or BigQuad deck to connect the servo

Reply:
yeah or just change it to 'via a prototype deck'. I think that should be clear too